tavern/_plugins/rest/response.py

```diff
@@ -1,7 +1,6 @@
 import json
-import traceback
 import logging
-import copy
+import traceback
```

Comment:
Not used?
```suggestion
```

Reply:
It's used in a couple places in that file